README.md

```diff
@@ -1 +1,158 @@
+
+## 🛠 Built With <a name="built-with"></a>
+Visual Studia Code, HTML/JavaScript linters, ES6, Webpack, API.
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+</details>
+
+<details>
+  <summary>JavaScript ES6</summary>
+</details>
+
+<details>
+  <summary>CSS</summary>
+</details>
+
+<details>
+  <summary>Webpack</summary>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Linters implementation]**
+- **[Single Page application]**
+- **[JavaScript Modules]**
+- **[Webpack framework]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://tajemouti.github.io/js-capstone-project/dist/)
+
+### Presentation <a name="presentation"></a>
+
+- [Presentation Link](https://drive.google.com/file/d/1-FwCiTOHKV4ZsV4Qn1ZfrypW0-0Da7RG/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Setup <a name="setup"></a>
+
+In order to use this project.. Clone this repository to your desired folder by pasting this command in your command line interface:
+
+  https://github.com/tajemouti/js-capstone-project.git
+
+### Prerequisites <a name="prerequisites"></a>
+
+  Node.js
+  ES6
+  Git
+  HTML/JavaScript linters
+  Webpack
+  Webpack sever
+  API
+
+### Install <a name="install"></a>
+
+Install dependencies:
+
+  npm install
+
+
+### Usage <a name="usage"></a>
+
+-Run: npm start
+
+
+### Run tests <a name="run-tests"></a>
+
+To test this application, run:
+
+  npm test
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@tajemouti](https://github.com/tajemouti)
+- Twitter: [@AjrassTajemouti](https://twitter.com/AjrassTajemouti)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ajrass)
+
+👤 **Author2**
+
+- GitHub: [@Bilal](https://github.com/BilalLiaquat7)
+- Twitter: [@](https://twitter.com/)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/muhammad-bilal-liaquat-87863390/)
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[A medium fedility wire-frame application]**
+- [ ] **[Secure deployment using GitHub Pages]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+Kindly give this project a star if you like it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](/LICENSE) license.
```

Comment:
- [ Optional ] I think it would be nice if you fix the License link on your readme file, I noticed that when I click on the link it leads to a `404` page not found error, a license serves to inform anyone who would like to use your project how it is Licensed. I would like to suggest that you add a file called `LICENSE` to the root directory of your project to fix this. You can use the content on this [link >> ](https://opensource.org/license/mit/) as the body of your License, remember to customize it.

## Error on link
![Screenshot from 2023-08-12 00-11-24](https://github.com/tajemouti/js-capstone-project/assets/16392046/5c86ec50-b312-465a-976e-a974a8a5d22f)